commands/stats.go

```diff
@@ -0,0 +1,101 @@
+package commands
+
+import (
+	"math"
+	"sort"
+
+	"github.com/pkg/errors"
+)
+
+// Stats is a struct that specifies the generated stats
+// Quantiles are calculated with Midpoint interpolation
+type Stats struct {
+	Mean      float64
+	Std       float64
+	Max       float64
+	Quantile3 float64
+	Median    float64
+	Quantile1 float64
+	Min       float64
+}
+
+//BuildStats generates a stats struct from an input float64 slice
+//It returns an error if the slice is empty
```

Comment:
These two lines are missing a space after the `//` to be consistent with the others (sorry, details stand out to me haha)